release/kubernetes-manifests.yaml

```diff
@@ -33,7 +33,7 @@ spec:
       terminationGracePeriodSeconds: 5
       containers:
       - name: server
-        image: gcr.io/google-samples/microservices-demo/emailservice:v0.2.4
+        image: gcr.io/google-samples/microservices-demo/*:v0.3.0
```

Comment:
Hmm, something went wrong with the `gsed` replacement. 
Looking into it now.

Reply:
Figured it out. I ran `./make-release-artifacts.sh` from the `hack/` directory.
I should've run it from this project's root directory. 
```
`./hack/make-release-artifacts.sh`
```